pkg/render/simple.go

```diff
@@ -40,7 +41,7 @@ func (r Simple) File(_ context.Context, fr *malcontent.FileReport) error {
 	})
 
 	for _, b := range bs {
-		fmt.Fprintf(r.w, "%s\n", b.ID)
+		fmt.Fprintf(r.w, "%s:%s%s\n", b.ID, strings.Repeat(" ", maxLength(bs)-len(b.ID)+1), strings.ToLower(b.RiskLevel))
```

Comment:
Can you just make this a single space between the ID and the risk level? Otherwise, we'll end up with the same inscrutable diff problem that markdown samples have today.

Reply:
Updated in [`8395a2a` (#516)](https://github.com/chainguard-dev/malcontent/pull/516/commits/8395a2a8d5c6557d247e5db4de853a4aefd0989c).